main.cpp

```diff
@@ -28,7 +28,19 @@ int main(){
       cout << "=" << num1*num2 << endl;
     }
     else if( input == "divide" ){
-      cout << "=" << num1/num2 << endl;
+      int remainder = num1%num2;
```

Comment:
Move this to the else statement, we don't want to do any unnecessary calculations if num2 == 0